app/og/route.tsx

```diff
@@ -0,0 +1,30 @@
+import { ImageResponse } from "next/server";
+// App router includes @vercel/og.
+// No need to install it.
+
+export const runtime = "edge";
+
+export async function GET() {
```

Comment:
Do you need this? I thought Next has a convention for OG images with just the file nam

Reply:
yeah but thats just for static images i think